samples/electron/metrics.js

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { Glean, _MetricTypes: { EventMetricType } } = require("glean");
```

Comment:
As for the QML PR, please remember that this is creating a new requirement for GleanJS: products will need at least a custom metric to use it.